.insomnia/Request/req_84c3c10ae54b4947b1bfae581f94db27.yml

```diff
@@ -0,0 +1,31 @@
+_id: req_84c3c10ae54b4947b1bfae581f94db27
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1660088304634
+created: 1660087332898
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/searches/{% response
```

Comment:
How does this look like? I suggest adding a screenshot for every PR.

Reply:
Like this:

![image](https://user-images.githubusercontent.com/9405286/183983676-4231f434-d70b-46cb-b3b1-22cd2d88cd85.png)

Reply:
So this basically deletes the same search that was created in the preceding endpoint?

Reply:
Yeah, pretty much. Isn't this preferred? The alternative would be to supply an id manually.

Reply:
No problem.